src/parse/asp/builtins.go

```diff
@@ -758,17 +759,28 @@ func subrepo(s *scope, args []pyObject) pyObject {
 		state.Config.Bazel.Compatibility = true
 		state.Config.Parse.BuildFileName = append(state.Config.Parse.BuildFileName, "BUILD.bazel")
 	}
+
+	isCrossCompile := s.pkg.Subrepo != nil && s.pkg.Subrepo.IsCrossCompile
+	if args[5] != None { // arg 5 is arch-string, for arch-subrepos.
+		var arch cli.Arch
+		givenArch := string(args[5].(pyString))
+		if err := arch.UnmarshalFlag(givenArch); err != nil {
+			log.Fatalf("Could not interpret architecture '%s' for subrepo '%'", givenArch, name)
```

Comment:
wow didn't know Github did this, but it has spotted that the second format parameter is missing the s. Cute!